.evergreen/evergreen.yml.in

```diff
@@ -1053,6 +1053,7 @@ buildvariants:
     tasks:
       - name: pkg_test_docker_centos7_rpm
       - name: pkg_test_docker_centos8_rpm
+      - name: pkg_test_docker_fedora34_rpm
```

Comment:
I noticed that this was missing from the list when I was editing the docker files.